optimum/intel/openvino/quantization.py

```diff
@@ -315,6 +315,31 @@ def quantize(
         else:
             raise TypeError(f"Unsupported model type: {type(self.model)}")
 
+    def _check_model_state(self, sub_model_names: List[str] = None):
+        message_template = (
+            "Couldn't apply optimization to the model because it was already compressed with config: {}. "
+            "To avoid this issue, set load_in_8bit=False in the from_pretrained method when using the optimum-intel API, "
```

Comment:
```suggestion
            "To avoid this issue, check that you set load_in_8bit=False or not using quantization_config at export in the .from_pretrained(),"
```